src/components/Ui/Text.vue

```diff
@@ -20,19 +20,3 @@ const textWithLinks = computed(() =>
 <template>
   <span v-html="textWithLinks" />
 </template>
```

Comment:
Maybe this entire component can be removed.
- It's name isn't super expressive like this.
- It supports only one [Autolinker option](https://github.com/gregjacobs/Autolinker.js#options) (truncate).
- It's only used in two places (Terms.vue and SpaceAbout.vue).
- In both cases it doesn't need to be a computed property.

The two "inline" replacements would look like this:

```vue
<span v-html="Autolinker.link(space.about)" />
```
and
```vue
<span v-html="Autolinker.link(getIpfsUrl, { truncate: 35 })" />
```

Plus the Autolinker import in both files. We could also use a wrapper function in helpers/utils. But a component seems a bit much to me. But in the end... if it works, it works.